recipes/libmikmod/all/patches/0001-CMakeLists.txt.patch

```diff
@@ -0,0 +1,63 @@
+--- "CMakeLists.txt"	2017-04-07 16:55:00.000000000 +0200
++++ CMakeLists.txt	2017-07-21 20:01:33.330845600 +0200
+@@ -25,6 +25,8 @@
+ 
+ PROJECT(libmikmod C)
+ CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
++include(../conanbuildinfo.cmake)
```

Comment:
I think it is more common for CCI to use a wrapping `CMakeLists.txt` than to include `conanbuildinfo.cmake` directly into upstream.